re2c/re2c.wake

```diff
@@ -36,7 +36,11 @@ def re2cOk Unit =
 def re2cReal file =
   def cpp = replace `\.re$` ".cpp" file.getPathName
   def cmdline = which "re2c", "-8", "--tags", "--no-generation-date", "--input-encoding", "utf8", file.getPathName, "-o", cpp, Nil
-  def result = job cmdline (file, Nil) | getJobOutput
+  def result =
+    makePlan cmdline (file, Nil)
+    | setPlanLabel "re2c {cpp}"
```

Comment:
This looks pretty different than the Label in https://github.com/sifive/wake/pull/454, so I just wanted to check that this basically has no bearing to the original use case introduced there.

The main practical difference I see is that in the other PR, the label was intended to mean something that code could match on. Here, by embedding the file paths of the inputs of the command into the label, you are defeating the ability to programmatically match based on string, except with substring matching.

The intent of the label in https://github.com/sifive/wake/pull/454 was that many instances of a rule would all share a label, so the label was not necessarily unique in a given run of Wake.

Reply:
I would say it depends how you use the label. Within the context of the wake build "re2c symbol.cpp" is a pretty useful key. As you note, you can still look them up using a RegExp or whatever.

Reply:
However, I do agree the intended use of this label is mostly aimed at improving the progress view of the build, as compared to the other PR.